bin/import_fisher.py

```diff
@@ -255,5 +260,25 @@ def _split_sets(filelist):
     )
 
 
+def get_sample_size(population_size):
```

Comment:
Same, I'm wondering if we should not have that sitting in https://github.com/mozilla/DeepSpeech/blob/385c8c769bc4aed5e6979a239591486c44f3471d/training/deepspeech_training/util/importers.py and be used from elsewhere